package.json

```diff
@@ -13,7 +13,7 @@
     "upload": "node ./scripts/commands.js upload",
     "update_patches": "node ./scripts/commands.js update_patches",
     "apply_patches": "node ./scripts/sync.js --run_hooks",
-    "start": "node ./scripts/commands.js start",
+    "start": "node ./scripts/commands.js start --enable-smart-tracking-protection",
```

Comment:
To-Do: Revert before merge. STP will be disabled by default

Reply:
If you're right